source/hands_on/coma.rst

```diff
@@ -71,6 +71,23 @@ The network interface COMA used is defined as follows:
 
 The Benchmark result of COMA in SMAC (Samvelyan et al. 2019), for StarCraft micromanagement problems, implemented in DI-engine is shown.
 
++---------------------+-----------------------------------------------------+
+| SMAC Map            | evaluation results                                  | 
++=====================+=====================================================+
+|                     |                                                     |
+|                     |                                                     |
+|                     |                                                     |
+|MMM                  |.. image:: images/benchmark/COMA_MMM.png             |
+|                     |                                                     |
+|                     |                                                     |
++---------------------+-----------------------------------------------------+
+|                     |                                                     |
+|                     |                                                     |
+|3s5z                 |                                                     |
+|                     |.. image:: images/benchmark/COMA_3s5z.png            |
+|                     |                                                     |
++---------------------+-----------------------------------------------------+
+
 References
 ----------
 
```

Comment:
reference has a standard format, if you can't find it, please contact me (hs)
e.g.:https://github.com/opendilab/DI-engine-docs/blob/main/source/hands_on/r2d3_zh.rst